shared/src/business/useCases/processStreamRecords/processCaseEntries.test.js

```diff
@@ -85,6 +85,19 @@ describe('processCaseEntries', () => {
     expect(caseDocketEntryMappingRecord).toEqual('CaseDocketEntryMapping');
   });
 
+  it('should index a case message mapping record in order to create a parent-child relationship between a case and a message', async () => {
```

Comment:
wonderfully named tests so far - clear as to what the intent is in the code. 👍 ⚡